tobac/utils/general.py

```diff
@@ -553,3 +554,129 @@ def combine_tobac_feats(list_of_feats, preserve_old_feat_nums=None):
 
     combined_df = combined_df.reset_index(drop=True)
     return combined_df
+
+
+@internal_utils.irispandas_to_xarray
+def transform_feature_points(
+    features,
+    new_dataset,
+    latitude_name="auto",
+    longitude_name="auto",
+    max_time_away=datetime.timedelta(minutes=1),
+    max_space_away=20000,
+):
+    """Function to transform input feature dataset horizontal grid points to a different grid.
+    The typical use case for this function is to transform detected features to perform
+    segmentation on a different grid.
+
+    The existing feature dataset must have some latitude/longitude coordinates associated
+    with each feature, and the new_dataset must have latitude/longitude available with
+    the same name.
```

Comment:
I've encountered this issue as well. Ideally we should modify the iris cube so that it has the same coordinate names as the xarray dataset (i.e. the short name rather than standard name). Mostly this is so that there is continuity in the coordinate naming across conversions, but there are also some issues that can arise with multiple coordinates having the same standard name

I think that this is beyond the scope of this PR, but something we should address for the 1.6 release

Reply:
I agree, that makes sense!

Reply:
I have hopefully addressed this the best we can in the docs; let me know if more clarification is needed.

Reply:
Looks good in the documentation. And as discussed, we will address this issue in 1.6!